test/CMakeLists.txt

```diff
@@ -183,21 +183,21 @@ if(HAVE_LIBUDEV_H)
 endif()
 
 include("${SDL3_SOURCE_DIR}/cmake/FindFFmpeg.cmake")
-if(FFmpeg_FOUND AND FFmpeg_AVCODEC_VERSION VERSION_GREATER_EQUAL "60")
+if(FFmpeg_FOUND AND (FFmpeg_AVCODEC_VERSION STREQUAL "" OR FFmpeg_AVCODEC_VERSION VERSION_GREATER_EQUAL "60"))
```

Comment:
Why are you allowing an empty `FFmpeg_AVCODEC_VERSION` string here?

Reply:
I don't know the reason, but right now the CMake version detection depends on pkg-config.
We should probably extract it from the headers directly so it also works on windows where pkg-config is not available by default.

Reply:
Well, it seems very wrong as it is now.

Reply:
What is the ffmpeg / avcodec function you want to use which is available only after a certain version? Maybe check that instead, either through the header or by attempting to link to it?

Reply:
This is the error that you get if your ffmpeg version is too old:
```
/home/slouken/projects/SDL/test/testffmpeg.c:658:93: error: no member named 'ch_layout' in 'struct AVCodecParameters'
    SDL_Log("Audio stream: %s %d channels, %d Hz\n", avcodec_get_name(codec->id), codecpar->ch_layout.nb_channels, codecpar->sample_rate);
                                                                                  ~~~~~~~~  ^
/home/slouken/projects/SDL/test/testffmpeg.c:681:53: error: no member named 'ch_layout' in 'struct AVCodecParameters'
    SDL_AudioSpec spec = { SDL_AUDIO_F32, codecpar->ch_layout.nb_channels, codecpar->sample_rate };
                                          ~~~~~~~~  ^
/home/slouken/projects/SDL/test/testffmpeg.c:753:70: error: no member named 'ch_layout' in 'struct AVFrame'
        SDL_AudioSpec spec = { GetAudioFormat(frame->format), frame->ch_layout.nb_channels, frame->sample_rate };
                                                              ~~~~~  ^
/home/slouken/projects/SDL/test/testffmpeg.c:756:20: error: no member named 'ch_layout' in 'struct AVFrame'
        if (frame->ch_layout.nb_channels > 1 && IsPlanarAudioFormat(frame->format)) {
            ~~~~~  ^
4 errors generated.
```

Reply:
That's a good idea! We'd probably want to check AVFrame, since that's where we use it in the code.

@madebr, do you want to add your magic?

Reply:
I applied sezero's suggestion in ee53e4d319098efec46dea198bd32c8dd868cf9b, testing for `ch_layout` in `AVFrame`.
ebfbd7327b90211a945ecdf5018cacd2c54dd014 lets it make use of SDL_test for parsing arguments, and tracking memory.
No leaks were detected!

Reply:
Looks great, thank you!